tests/test_install.py

```diff
@@ -0,0 +1,103 @@
+import importlib
+import subprocess
+import sys
+
+import pytest
+
+import audeer
+
+
+PACKAGE = 'PyYaml'
+MODULE = 'yaml'
+
+
+def uninstall():
+    subprocess.run(
+        [
+            sys.executable,
+            '-m',
+            'pip',
+            'uninstall',
+            '--yes',
+            PACKAGE,
+        ],
+        stdout=subprocess.DEVNULL,
+        stderr=subprocess.DEVNULL,
+    )
+    # remove from module cache
```

Comment:
Would this also work in general?
If yes we would have the possibility to first remove the package if a wrong version of it is installed.